src/Rector/ClassMethod/MergeMethodAnnotationToRouteAnnotationRector.php

```diff
@@ -64,6 +73,13 @@ class DefaultController extends Controller
     public function show($id)
     {
     }
+
+    /**
+     * @Route("/post/{id}", methods={"POST"})
+     */
+    public function post($id)
+    {
+    }
```

Comment:
Could you keep this sample as small as it was? It is used to generate documentation and it should contain small code samples to make the whole docs easier to read :+1: